homeassistant/components/notion/__init__.py

```diff
@@ -216,7 +247,7 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
 @callback
 def _async_register_new_bridge(
-    hass: HomeAssistant, bridge: Bridge, entry: ConfigEntry
+    hass: HomeAssistant, entry: ConfigEntry, bridge: Bridge
```

Comment:
FYI, this is for consistency and similarity to the rest of the integration: `hass` first + HASS-related objects before Notion-related objects.